src/header.py

```diff
@@ -124,9 +142,9 @@ class NotThisMethod(Exception):
 HANDLERS: Dict[str, Dict[str, Callable]] = {}
 
 
-def register_vcs_handler(vcs, method):  # decorator
+def register_vcs_handler(vcs: str, method: str) -> Callable:  # decorator
```

Comment:
I agree this would be a lot better than what I have here, however I'm considering doing even better than this in a future patch. I didn't want to tie up this larger improvement with further improving this decorator (mainly since *users* of the library will be unlikely to ever need to touch this decorator.)

That said, if you feel strongly I can do that extra improvement as part of this PR.

Reply:
I don't feel strongly.